src/main/java/com/zextras/operations/OperationResult.java

```diff
@@ -0,0 +1,33 @@
+package com.zextras.operations;
+
+/**
+ * Class that provides information about the result of operations you perform.
+ */
+public class OperationResult {
+
+  private final String name;
+  private final int intValue;
+  private final String stringRepresentation;
+
+  /**
+   * @param name     plain message of an operation which is clear to you.
+   * @param intValue plain operation code which is clear to you.
+   */
+  public OperationResult(String name, int intValue) {
+    this.name = name;
+    this.intValue = intValue;
+    this.stringRepresentation = intValue + " (" + name + ')';
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public int getIntValue() {
+    return intValue;
+  }
+
+  public String getStringRepresentation() {
+    return stringRepresentation;
+  }
```

Comment:
Override `toString` method instead of implementing a new one (which does the same)
```suggestion
  @Override
  public String toString() {
    return stringRepresentation;
  }
```

Reply:
done!